advocacy_docs/partner_docs/VisionXGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: 'Configuring the Integration'
```

Comment:
Is there anything we need to configure / setup between EDB and SIB Visions?  Or is the integration just when you are running and using SIB Visions VisionX?

If the integration is just in use then for config we should do something like

Install EDB according to your needs and keep track of the following items 
username/password for a database user.
port number / url 
etc.  
What will be need when using the integration later on.
or we could say connect to your database and create the appropriate users that will access DB from sib visions.

for config on VisionX we can say install according to install docs and make sure that the system running VisionX  can communicate with the DB system.